### PR TITLE
 add PHP rules for Symfony (part 2) 

### PR DESCRIPTION
**Pull request by inkz**

### CODE REVIEW

File: php/lang/security/non-literal-header.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+rules:
+- id: non-literal-header
+  patterns:
+  - pattern: header(...)
+  - pattern-not: header("...",...)
+  message: >-
+    Using user input when deleting files with `unlink()` is potentially dangerous. A malicious actor could use this to modify
```
Comment (minusworld) [inline php/lang/security/non-literal-header.yaml]: I think this message is incorrect?

**Review by minusworld — CHANGES_REQUESTED** [review]
Rule message

File: php/lang/security/non-literal-header.yaml (hunk 1 above)
Comment (inkz) [reply php/lang/security/non-literal-header.yaml]: Ooops :sweat_smile: thanks a lot for finding it! updated :+1:

File: php/lang/security/ldap-bind-without-password.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+$ldapconn = ldap_connect("foo.com");
+
+// ruleid: ldap-bind-without-password
+$ldapbind = ldap_bind($ldapconn);
+
+// todoruleid: ldap-bind-without-password
+LDAP_BIND($ldapconn, "username");
```
Comment (minusworld-gha-demo) [inline php/lang/security/ldap-bind-without-password.php:9]: @inkz What's would it take for the PHP parser to match this test case?

**Review by minusworld-gha-demo — APPROVED** [review]

**Review by minusworld — APPROVED** [review]

**Pull request merged**